tools/wpt/browser.py

```diff
@@ -579,8 +579,12 @@ def install_webdriver(self, dest=None, channel=None):
         raise NotImplementedError
 
     def version(self, binary=None, webdriver_binary=None):
-        return None
-
+        command = "(Get-AppxPackage Microsoft.MicrosoftEdge).Version"
+        try:
+            return call("powershell.exe", command).strip()
+        except subprocess.CalledProcessError:
```

Comment:
From the failures it appears that OSError is thrown when powershell.exe doesn't even exist. That hasn't happened for safari because of the `webdriver_binary` check in place.

Reply:
Catching `OSError` too worked.